src/openvic-simulation/pop/Pop.hpp

```diff
@@ -346,3 +371,6 @@ namespace OpenVic {
 		bool parse_scripts(DefinitionManager const& definition_manager);
 	};
 }
+#ifndef DO_FOR_ALL_TYPES_OF_POP_INCOME
```

Comment:
The define for keeping this macro needs to have a different name from the macro, otherwise it gets overwritten by the macro (producing a compiler warning but still working) and then doesn't get undefined since `#ifndef DO_FOR_ALL_TYPES_OF_INCOME` always fails.
Change it to `KEEP_DO_FOR_ALL_TYPES_OF_INCOME` here and at the top of the cpp file